curriculum/challenges/english/01-responsive-web-design/applied-visual-design/use-the-css-transform-scale-property-to-scale-an-element-on-hover.english.md

```diff
@@ -10,6 +10,8 @@ videoUrl: 'https://scrimba.com/c/cyLPJuM'
 The <code>transform</code> property has a variety of functions that lets you scale, move, rotate, skew, etc., your elements. When used with pseudo-classes such as <code>:hover</code> that specify a certain state of an element, the <code>transform</code> property can easily add interactivity to your elements.
 Here's an example to scale the paragraph elements to 2.1 times their original size when a user hovers over them:
 <blockquote>p:hover {<br>&nbsp;&nbsp;transform: scale(2.1);<br>}</blockquote>
+   
+  **NOTE:** Applying a transform to a <code>div</div> element will also affect any child elements contained in the div.
```

Comment:
```suggestion
  **NOTE:** Applying a transform to a <code>div</code> element will also affect any child elements contained in the div.
```

Reply:
@thecodingaviator Good catch!

Reply:
Ah, yes, just noticed it. I've been typing many divs. Thank you for changing it.

Reply:
@leebut You still need to change it.  @thecodingaviator just made a suggestion.

Reply:
Okay. Done. Thank you.